app/controllers/devise_token_auth/registrations_controller.rb

```diff
@@ -28,68 +21,64 @@ def create
           status: 'error',
           data:   @resource.as_json,
           errors: [I18n.t("devise_token_auth.registrations.missing_confirm_success_url")]
-        }, status: 403
+        }, status: :bad_request
       end
 
       # if whitelist is set, validate redirect_url against whitelist
-      if DeviseTokenAuth.redirect_whitelist
-        unless DeviseTokenAuth.redirect_whitelist.include?(redirect_url)
-          return render json: {
-            status: 'error',
-            data:   @resource.as_json,
-            errors: [I18n.t("devise_token_auth.registrations.redirect_url_not_allowed", redirect_url: redirect_url)]
-          }, status: 403
-        end
+      if DeviseTokenAuth.redirect_whitelist &&
+         !DeviseTokenAuth.redirect_whitelist.include?(redirect_url)
+        return render json: {
+          status: 'error',
+          data:   @resource.as_json,
+          errors: [I18n.t("devise_token_auth.registrations.redirect_url_not_allowed", redirect_url: redirect_url)]
+        }, status: :forbidden
       end
 
```

Comment:
Same as above, based on the [ruby styleguide](https://github.com/bbatsov/ruby-style-guide#begin-implicit)